pages.it/common/git-grep.md

```diff
@@ -0,0 +1,25 @@
+# git-grep
+
+> Cerca stringhe nello storico dei file tracciati nel repository.
+> Supporta molti degli stessi flag accettati dal `grep` tradizionale.
+> Maggiori informazioni: <https://git-scm.com/docs/git-grep>.
+
+- Cerca una stringa nei file tracciati:
+
+`git grep {{stringa_ricercata}}`
+
+- Cerca una stringa nei file tracciati che soddisfano un dato pattern:
+
+`git grep {{stringa_ricercata}} -- {{file_pattern}}`
```

Comment:
```suggestion
`git grep {{stringa_ricercata}} -- {{file_glob_pattern}}`
```
I would leave "glob" here since it is a well known kind of path pattern.